### PR TITLE
fix(client): Code solution modal on timeline

### LINKED ISSUE #45948: Code solution modal on timeline is more narrow than others
### Describe the Issue

The modal for viewing code solutions on the timeline page is more narrow than all the other modals.

### Steps to Reproduce

1. Go to https://www.freecodecamp.dev/fccb1
2. Click view -> code for one of the new RWD cert projects
3. See narrow modal
4. Close modal -> click view -> project
5. See wider modal
6. Go to https://www.freecodecamp.dev/certification/fccb1/responsive-web-design
7. View the same two modals, they're both the wide version

### Expected behavior

All of them to be the wide version

### PR DESCRIPTION
**Pull request by CAPTAINCOBRA**
First open source commit
I added a size attribute of large to the modal so that its size increases on lg display. The rest of the behavior remains same. 
I have tested it in my local code.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45948

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
This looks good to me. Modal seems to be the same size as the others.

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **AWESOME!** :sparkles: :sparkles: 

LGTM.

Thanks for this pull request, @CAPTAINCOBRA, and for contributing to the code-base for the first time.

⚠️ Please note a few key things (so that maintainers take your contributions seriously):

1. Ensure that you read and fill out the pull-request checklist. It's there for a reason.
2. You have made the PR from the `main` branch of your fork, which will soil it. The next time you make a PR from the same fork of yours, you will have unwanted changes there. I would advise deleting the fork once this is merged if you are interested in making contributions in the future.
3. Avoid using GitHub to edit your changes directly because with copy-pasting, you may miss things between your local & the GitHub UI. Instead, you should make commits from your machine OR, even better use the one-click setup with GitPod.
4. Use a meaningful PR title next time. It saves everyone time.

Our contributing guidelines explain all this in great detail. You should definitely give them a read.

We will be looking forward to more contributions in the future.

Cheers and happy contributing!

**Pull request merged**
